### PR TITLE
Substitute about:debugging#/runtime/this-firefox for about:debugging

### PR DESCRIPTION
**Pull request by guest271314**
Extensions are loaded at about:debugging#/runtime/this-firefox not about:debugging

### CODE REVIEW

**Comment by Rob--W** [comment]
This PR does two independent things, with the second one being unrelated to the first. Could you split the PR?

As for the first commit, I think that it would be better to suggest to click on the button to debug "This ...(browser)" instead of typing that full URL.

**Comment by guest271314** [comment]
@Rob--W Can you kindly review the split PR's?

File: native-messaging/README.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ To assist in troubleshooting on Windows, there is a script called `check_config_
 
 ## Testing the example ##
 
-Then just install the add-on as usual, by visiting about:debugging, clicking "Load Temporary Add-on", and selecting the add-on's "manifest.json".
+Then just install the add-on as usual, by visiting `about:debugging#/runtime/this-firefox` or click on the button to debug "This Firefox" , clicking "Load Temporary Add-on", and selecting the add-on's "manifest.json".
```
Comment (rebloor) [inline native-messaging/README.md:28]: In nightly the menu option is "This Nightly"
```suggestion
First, install the add-on. Visit `about:debugging#/runtime/this-firefox` or, from `about:debugging` click "This Firefox" (or "This Nightly" in the Nightly version of Firefox), click "Load Temporary Add-on", and open the add-on's "manifest.json".
```

**Review by rebloor — APPROVED** [review]

**Comment by rebloor** [comment]
Thanks @guest271314

**Pull request merged**